resources/assets/components/pages/AccountPage/Subscriptions/CancelEmailSubscription.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import gql from 'graphql-tag';
+import { useQuery, useMutation } from '@apollo/react-hooks';
+
+import Spinner from '../../../artifacts/Spinner/Spinner';
+
+const EMAIL_SUBSCRIPTION_STATUS = gql`
+  query EmailSubscriptionStatus($userId: String!) {
+    user(id: $userId) {
+      id
+      emailSubscriptionStatus
+    }
+  }
+`;
+const EMAIL_SUBSCRIPTION_STATUS_MUTATION = gql`
+  mutation EmailSubscriptionStatus(
+    $userId: String!
+    $emailSubscriptionStatus: Boolean!
+  ) {
+    updateEmailSubscriptionStatus(
+      id: $userId
+      emailSubscriptionStatus: $emailSubscriptionStatus
+    ) {
+      id
+      emailSubscriptionStatus
+    }
+  }
+`;
+const CancelEmailSubscription = props => {
+  const userId = props.user.id;
+  const options = { variables: { userId } };
+
+  const { data, loading, error } = useQuery(EMAIL_SUBSCRIPTION_STATUS, options);
+  const [updateEmailSubscriptionStatus] = useMutation(
+    EMAIL_SUBSCRIPTION_STATUS_MUTATION,
+    options,
+  );
+
+  if (error) {
+    return <p>Something went wrong!</p>;
+  }
+
+  if (loading) {
+    return <Spinner />;
+  }
+  return (
```

Comment:
Nitpick: Let's include a newline between this conditional and the `return` to give it some breathing room:

```suggestion

  return (
```